manifest.json

```diff
@@ -2,20 +2,21 @@
   "name": "Tiny Tiny RSS",
   "id": "ttrss",
   "packaging_format": 1,
-  "requirements": {
-    "yunohost": ">> 2.3.12.1"
-  },
   "description": {
     "en": "A PHP and Ajax feed reader",
     "fr": "Un lecteur de flux en PHP et Ajax"
   },
   "url": "http://tt-rss.org",
-  "version": "17.1",
+  "license": "GPL-3",
+  "version": "17.4",
```

Comment:
Here shouldn't we put the commit id instead?

Reply:
Euh, yes if you want.
Why not.
Maybe it's not really clear for the final user ?